InnoTopicWebsite/src/app/countries/country-flag/country-flag.component.css

```diff
@@ -0,0 +1 @@
+/* No CSS *//*# sourceMappingURL=country-flag.component.css.map */
```

Comment:
Done for all components except for 'app/shares/company-logo/company-logo.component.css' which even though is empty, is directly referenced by its component. I assume is the boiler plate for future code, so I'm leaving that one unless specified otherwise. All other .css files has been safely removed.

Reply:
Thanks. True. company-logo.component.css is different :).